opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -67,6 +67,7 @@
 #include <opm/parser/eclipse/EclipseState/Tables/SwofTable.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableContainer.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/WatvisctTable.hpp>
+#include <opm/parser/eclipse/EclipseState/Tables/DensityTable.hpp>
```

Comment:
> is DENSITY a table keyword? As far as I can remember, there's only one entry per record and nothing is interpolated?

One (per phase) entry per record, which corresponds to regions. No interpolation as far as I'm aware.

It's as @joakim-hove points out an edge case - it's very similar in structure to a lot of the other tables, and there might be more properties soon that are more similar to DENSITY than anything else. We'll see, maybe it'll soon be time for `properties` in addition to `tables`, but there's plenty of time for that later.

I'm happy to move it though if we find a better home.